### PR TITLE
card-mod-defaults

### PR DESCRIPTION
**Pull request by ethn**

### CODE REVIEW

File: card/gem_mod/card-mod-ace_editor/card-mod-ace_editor.gemspec [hunk 1]
```diff
@@ -20,4 +20,5 @@ Gem::Specification.new do |s|
   s.required_ruby_version = ">= 2.3.0"
   s.metadata = { "card-mod" => "ace_editor" }
   s.add_runtime_dependency "card-mod-edit", Versioning.simple
+  s.add_runtime_dependency "card", Versioning.card
```
Comment (ethn) [inline card/gem_mod/card-mod-ace_editor/card-mod-ace_editor.gemspec:23]: I guess this is redundant...

File: card/mod/Modfile [hunk 2]
```diff
@@ -23,10 +23,5 @@ mod 'navbar'
 mod 'google_analytics'
 gem_mod 'edit'
```
Comment (GerryG) [inline card/mod/Modfile:24]: We need the Modfile to put this before bootstrap? Seems like a small issue we should fix when code gets moved around and into more gems. then this ordering is in the default pack gem.

File: card/gem_mod/card-mod-ace_editor/card-mod-ace_editor.gemspec (hunk 1 above)
Comment (GerryG) [reply card/gem_mod/card-mod-ace_editor/card-mod-ace_editor.gemspec:23]: right, if you depend on another card-mod-gem, it likely already has card in the chain of deps. So either on card or another card-mod- is enough.

**Comment by GerryG** [comment]
If it builds and tests right, all the specs, etc. then it looks good.

File: card/mod/Modfile (hunk 2 above)
Comment (ethn) [reply card/mod/Modfile:24]: There is no need to worry about ordering in the pack.

`card-mod-bootstrap` will depend on `card-mod-edit`. Then worry not.

File: card/gem_mod/card-mod-ace_editor/card-mod-ace_editor.gemspec (hunk 1 above)
Comment (ethn) [reply card/gem_mod/card-mod-ace_editor/card-mod-ace_editor.gemspec]: fixed

**Pull request merged**